### ISSUE TITLE
enable/disable the boefje via a Json api call and implement this as progressive enhancement.

### ISSUE THREAD

**Issue by underdarknl** [minvws/nl-kat-coordination#3655]
The ultimate solution would be to enable/disable the boefje via a Json api call and implement this as progressive enhancement.

_Originally posted by @underdarknl in https://github.com/minvws/nl-kat-coordination/issues/3654#issuecomment-2401713988_

**Comment by underdarknl** [comment]
We'd need an api endpoint that allows the authenticated user to enable/disable the plugin, and then some Javascript to attach event handlers to the buttons.

**Comment by dekkers** [comment]
I think it would be better to use htmx than to write our own backend code for the endpoint and write our own javascript code each time we need to do something like this.

**Comment by underdarknl** [comment]
True, this is not a complex usecase, and fits the possibilities of htmx very well, while still being progressive enhancement if we do it properly.
